doc/src/records/accumulator.md

```diff
@@ -6,12 +6,26 @@ TODO: expand description and example
 
 ACCUMULATOR results must be reported using user-defined REPORTs or EXPORTs.  For example --
 
-    REPORT rpType=UDT rpFreq=Month rpDayBeg=Jan 1 rpDayEnd=Dec 31
-        REPORTCOL colHead="mon" colVal=$Month colWid=3
-        REPORTCOL colHead="Total" colVal=@Accumulator[ 1].M.acmSum colDec=0 colWid=10
-        REPORTCOL colHead="Average" colVal=@Accumulator[ 1].M.acmMean colDec=0 colWid=10
+    ACCUMULATOR "Zone Sensible Cooling Rate [Btu/h]" 
+      acmValue = @znRes[ 1].H.qscHvac // Hourly Output
```

Comment:
Is this the average Btu/h over the hour or the total Btu over the hour? The answer will impact the accuracy of the report values below.